opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -17,64 +17,38 @@
 # Docker registry
 ARG DOCKER_HUB_PUBLIC=docker.io
 
-#
-# Preliminary image that downloads Tomcat and makes some modifications
-#
-FROM ${DOCKER_HUB_PUBLIC}/opensuse/leap:latest AS builder
-RUN zypper -n install gzip tar xsltproc
-WORKDIR /wd
+#OpenSUSE Tomcat Base image
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:opensuse-tomcat-base-image-5.7.0-US590035a-SNAPSHOT AS base-image
```

Comment:
This image cannot be checked into main, its got a label base-image that's not used.

Reply:
After discussing this change with @michael-bryson, the above line should be represented along the lines of 
FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:opensuse-tomcat-jul-5.7.0-US590035a-SNAPSHOT AS builder so that we avoid static images being used, the ticket number would need removed in a separate commit just before merging i.e. would be: FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:opensuse-tomcat-base-image-5.7.0-SNAPSHOT AS base-image upon merging, however theres also been another issue with this command identified at https://github.com/CAFapi/opensuse-tomcat-image/pull/59#discussion_r1034826327